File: app/src/main/java/com/daviddf/geeklab/Experiments.java
package com.daviddf.geeklab;

import android.graphics.Bitmap;

public class Experiments {
    String  Imagen,Titulo, Url;

    public Experiments(){}

    public Experiments(String imagen, String titulo, String url) {
        Imagen = imagen;
        Titulo = titulo;
        Url = url;
    }

    public String getImagen() {
        return Imagen;
    }

    public void setImagen(String imagen) {
        Imagen = imagen;
    }

    public String getTitulo() {
        return Titulo;
    }

    public void setTitulo(String titulo) {
        Titulo = titulo;
    }

    public String getUrl() {
        return Url;
    }

    public void setUrl(String url) {
        Url = url;
    }

}

File: app/src/main/java/com/daviddf/geeklab/Experiments.kt
package com.daviddf.geeklab

import com.google.firebase.firestore.PropertyName

data class Experiments(
    @get:PropertyName("Imagen") @set:PropertyName("Imagen") var imagen: String? = null,
    @get:PropertyName("Titulo") @set:PropertyName("Titulo") var titulo: String? = null,
    @get:PropertyName("Url") @set:PropertyName("Url") var url: String? = null,
    @get:PropertyName("Tag") @set:PropertyName("Tag") var tag: String? = null
)
